pytorch_lightning/loops/dataloader/prediction_loop.py

```diff
@@ -110,8 +110,10 @@ def _on_predict_start(self) -> None:
         self.trainer.lightning_module.zero_grad()
 
         # hook
-        self.trainer.call_hook("on_predict_start")
-        self.trainer.call_hook("on_predict_epoch_start")
+        self.trainer._call_callback_hooks("on_predict_start")
+        self.trainer._call_lightning_module_hook("on_predict_start")
+        self.trainer._call_callback_hooks("on_predict_epoch_start")
```

Comment:
nit: a newline would be nice here

```suggestion

        self.trainer._call_callback_hooks("on_predict_epoch_start")
```